app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  belongs_to :users
```

Comment:
- Good job done so far, but there are some changes needed to be made, A post should belong to a `user` not `users`. Please fix this 🙏